_data/nav copy.yml.txt

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
What does this file do? is there a "space" in it's name?

Reply:
This file is for the faq and other pages links, which is already done but the data is not finished. Kept this file so as to retrieve the same once data is updated.

Reply:
This pr should not contain anything to do with the faq page. That is a separate pr on a separate branch.

Reply:
Ok, I will remove the file and push again.